Modules/socketmodule.c

```diff
@@ -2094,11 +2109,13 @@ getsockaddrarg(PySocketSockObject *s, PyObject *args,
                 return 0;
 #else  /* __NetBSD__ || __DragonFly__ */
             _BT_HCI_MEMB(addr, family) = AF_BLUETOOTH;
-            if (!PyArg_ParseTuple(args, "i", &_BT_HCI_MEMB(addr, dev))) {
+            unsigned short dev = _BT_HCI_MEMB(addr, dev);
```

Comment:
Where is it defined?

Reply:
There's a bunch of `define` at the top of the file:

```c
#if (defined(HAVE_BLUETOOTH_H) || defined(HAVE_BLUETOOTH_BLUETOOTH_H)) \
     && !defined(__NetBSD__) && !defined(__DragonFly__)
#define USE_BLUETOOTH 1
#if defined(__FreeBSD__)
...
#define _BT_HCI_MEMB(sa, memb) ((sa)->hci_##memb)
...
#elif defined(__NetBSD__) || defined(__DragonFly__)
...
#define _BT_HCI_MEMB(sa, memb) ((sa)->bt_##memb)
#else
...
#define _BT_HCI_MEMB(sa, memb) ((sa)->hci_##memb)
...
#endif
#endif
```

But AFAICT, the `elif defined(__NetBSD__) || defined(__DragonFly__)` is not possible at all since we're still in the big `#if` where we want `!defined(__NetBSD__) && !defined(__DragonFly__)`. So indeed, it looks like the macro wouldn't be defined here.